app/ts/components/Transfer.tsx

```diff
@@ -32,30 +32,18 @@ export const Transfer = ({ provider, blockInfo }: { provider: Signal<ProviderSto
 	})
 
 	function validateAddressInput(event: JSX.TargetedEvent<HTMLInputElement>) {
-		try {
-			addressInput.value = getAddress(event.currentTarget.value.toLowerCase())
-		} catch (e) {
-			console.error(e)
-			addressInput.value = undefined
-		}
+		const value = event.currentTarget.value.toLowerCase().trim()
```

Comment:
validateRecipientInput and validateAddressInput seem like the same function, could just use one that validates address

Reply:
Event handler for different input box.